hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMStorageConfig.java

```diff
@@ -76,6 +98,11 @@ protected Properties getNodeProperties() {
     }
     Properties scmProperties = new Properties();
     scmProperties.setProperty(SCM_ID, scmId);
+    String scmNodeInfo = getScmNodeInfo();
+    if (scmNodeInfo != null) {
+      // FOR NON-HA setup, SCM_NODES can be null
```

Comment:
If we add nodeInfo for non-HA, this can be avoided.

Reply:
I guess, we need to handle upgrades if we use this for non-HA cluster. Not sure we should this right away.

Reply:
Fine with me, If we want to revisit this later.